File: Code/Disney/disney.xBandController/src/linux/IDMSLib/src/main/java/com/disney/xband/idms/lib/model/PartyMember.java
package com.disney.xband.idms.lib.model;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
public class PartyMember
{
	
	private long guestId;
	private boolean isPrimary;
	private LinkCollection links;

	
	@XmlElement(name="guestId")
	public long getGuestId() {
		return guestId;
	}
	
	@XmlElement(name="guestId")
	public void setGuestId(long guestId) {
		this.guestId = guestId;
	}
	
	@XmlElement(name="isPrimary")
	public boolean isPrimary() {
		return isPrimary;
	}
	
	@XmlElement(name="isPrimary")
	public void setPrimary(boolean isPrimary) {
		this.isPrimary = isPrimary;
	}

	@XmlElement(name="links")
	public LinkCollection getLinks() {
		return links;
	}

	@XmlElement(name="links")
	public void setLinks(LinkCollection links) {
		this.links = links;
	}



}

File: Code/Disney/disney.xBandController/src/windows/Test/Disney.xBand.IDMS/Parties/PartyMember.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Runtime.Serialization;
using Disney.xBand.IDMS.Links;

namespace Disney.xBand.IDMS.Parties
{
    [DataContract]
    public class PartyMember
    {
        [DataMember(Name = "guestId")]
        public long GuestID { get; set; }

        [DataMember(Name = "isPrimary")]
        public bool IsPrimary { get; set; }

        [DataMember(Name = "links")]
        private LinkCollection Links { get; set; }
    }
}
